config/i18n.json

```
@@ -928,7 +928,7 @@
     "ShowNewItems": "Show a red dot on new items",
     "ExpandSingleCharacter": "Show all characters",
     "SingleCharacter": "Only show current character",
```

```suggestion
    "SingleCharacter": "Single-Character Mode",
```
i'd be fine with promoting "Single-Character Mode" to the actual name of the feature.
the current separateness and colon structure don't really make sense.

---

I could go either way. Single character mode maybe could be interpreted in some weird way.

---

Single-Character View?

---

Yeah that's clearer